internal/server/gcs/models.go

```diff
@@ -8,7 +8,13 @@ import (
 
 // BlobStorageClient This is used in service
 type BlobStorageClient interface {
-	ListTopicDates(bucketInfo BucketInfo) (map[string]map[string]int64, error)
+	ListTopicDates(bucketInfo BucketInfo) ([]TopicMetaData, error)
+}
+
+type TopicMetaData struct {
```

Comment:
you can use model from here
https://github.com/goto/dex/blob/main/generated/models/dlq.go#L20
we define our domain model via swagger and generated there.

Defining the model in gcs package and use it as response might not seem scalable.
Response might be enriched in the future, but it may not come from gcs.